components/custom-text-input.tsx

```diff
@@ -0,0 +1,11 @@
+import { blueDarkAlt } from '@pxblue/react-native-themes';
+import React from 'react';
+import { TextInput, useTheme } from 'react-native-paper';
+
+export const MyCustomTextInput: typeof TextInput = (props) => {
+    const theme = useTheme(props.theme);
+    const darkTheme = JSON.parse(JSON.stringify(blueDarkAlt));
+    darkTheme.colors.primary = theme.colors.primary;
```

Comment:
What is this for?

Reply:
This is a theme mashup. We're using the blueDarkAlt theme for the background colors but still need the primary color from 'blueDark' to make sure the outline for the textInput is blue[200] instead of the blue[500] defined in the alt theme.